apps/perf-test-react-components/tsconfig.json

```diff
@@ -1,13 +1,13 @@
 {
   "extends": "../../tsconfig.base.json",
   "compilerOptions": {
-    "target": "es5",
+    "target": "ES2019",
     "outDir": "lib",
     "module": "commonjs",
     "jsx": "react",
     "experimentalDecorators": true,
     "preserveConstEnums": true,
-    "lib": ["ES2015", "DOM"],
+    "lib": ["ES2019", "DOM"],
```

Comment:
@TristanWatanabe, @Hotell - any reason why I should not bump these?
The reason is I use `Object.entries()` in `tokens` package.

Reply:
I don't see any issues bumping this. All other v9 related packages use ES2019